tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/02_statuses/returnStatuses/01_CRUDOrderReturnStatus.ts

```diff
@@ -94,7 +94,7 @@ describe('BO - Shop Parameters - Order Settings - Statuses : Create, update '
       await testContext.addContextItem(this, 'testIdentifier', 'createOrderStatus', baseContext);
 
       const textResult = await addOrderReturnStatusPage.setOrderReturnStatus(page, createOrderReturnStatusData);
-      await expect(textResult).to.contains(statusesPage.successfulUpdateMessage);
+      await expect(textResult).to.contains(statusesPage.successfulCreationMessage);
```

Comment:
About the auto tests, if you reintroduce the legacy controller it may be too soon to adapt them as well